posts/2021-09-03-210010-beta.adoc

```diff
@@ -0,0 +1,237 @@
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Java 17 beta support, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.10-beta provides long-awaited Java 17 beta support, offering a lot of new enhancements and functionality. Also included are updates to MicroProfile Context Propagation, meaning that you can now use MicroProfile Context Propagation within your Jakarta EE 9 applications. This beta also offers JWE token format support for OpenID Connect Client and Social Media Login, as well as Jakarta EE 9 support for the remaining Open Liberty value-add features.
+blog_description: Open Liberty 21.0.0.10-beta provides long-awaited Java 17 beta support, offering a lot of new enhancements and functionality. Also included are updates to MicroProfile Context Propagation, meaning that you can now use MicroProfile Context Propagation within your Jakarta EE 9 applications. This beta also offers JWE token format support for OpenID Connect Client and Social Media Login, as well as Jakarta EE 9 support for the remaining Open Liberty value-add features.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Java 17 beta support, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta
+Ryan Storey <https://github.com/Ryan Storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 21.0.0.10-beta provides long-awaited Java 17 beta support, offering a lot of new enhancements and functionality. Also included are updates to MicroProfile Context Propagation, meaning that you can now use MicroProfile Context Propagation within your Jakarta EE 9 applications. This beta also offers JWE token format support for OpenID Connect Client and Social Media Login, as well as Jakarta EE 9 support for the remaining Open Liberty value-add features.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<java17, Java 17 Beta Support>>
+* <<mp_cp, MicroProfile Context Propagation 1.3>>
+* <<value_add, Jakarta EE 9 support for the remaining Open Liberty value-add features>>
+* <<jwe, JWE support for OpenID Connect Client and Social Media Login>>
```

Comment:
JWE should be the second item in the blog, so move it up here and the actual section below to follow java17.